draft-packages/button/KaizenDraft/Button/styles.scss

```diff
@@ -365,11 +365,11 @@ $caButton-verticalPaddingForm: calc(
   }
 
   &:not(:first-child) {
-    @include ca-margin($start: 0.5em);
+    @include ca-margin($start: 0.5em, $end: null);
```

Comment:
I did this because there was an issue when a third node was added to Button: `ca-margin($start: 0.5em` adds `$end: 0` by default which as overriding the `not(:last-child)` declaration and causing the second middle element to lose it's padding (as it's neither the last nor the first child, so both rules apply) . This change does not add any regressions (see Chromatic build)

Reply:
That's `null` working as intended there 👌